docs/compiler.md

```diff
@@ -32,11 +32,12 @@ a dialect of Go rather than a complete port of the language:
    is supported; type assertion panics if value can't be asserted to the desired type, therefore
    it's up to the programmer whether assert can be performed successfully.
  * type aliases including the built-in `any` alias are supported.
+ * `min()` and `max()` are supported for all types satisfying `cmp.Ordered`,
+   except `uintptr`, `float32`, `float64` and `string`.
```

Comment:
It's not true, because there might be user-defined `cmp.Ordered` types, whereas our compiler supports only integers.

And place it at L39, as it was before, for more clear git history of the file.